src/components/Button/Button.module.css

```diff
@@ -21,4 +21,4 @@
 
 .button button:hover {
 	background-color: var(--button-hover-color);
-}
+}
```

Comment:
why?

Reply:
what do you mean?

Reply:
I'm talking about removing the new lines at the end of the files